lms/djangoapps/grades/grade_utils.py

```diff
@@ -211,4 +217,7 @@ def get_scores(usage_key, user_ids=None):
             'modified': row.modified,
             'state': row.state,
         }
+        last_override = row.scoreoverrider_set.latest('created')
+        if last_override:
+            scores[row.student_id]['who_last_graded'] = last_override.user_id
```

Comment:
Should this actually be `user.id`, since the ScoreOverride models has a `user` field?

Reply:
`user.id` will do a query for the auth_user. In this case, none of the other information on the user is necessary.